### ISSUE TITLE
Fine-tune for a Domain Text Generation

### ISSUE THREAD

**Issue by Aminolroaya** [microsoft/BioGPT#111]
Hi I want to fine-tune BioGPT in HuggingFace for text generation in specific domain. I am using BioGptForCausalLM.from_pretrained("microsoft/biogpt")

When I train the model using Trainer.Train(), I get the error "The model did not return a loss from the inputs, only the following keys: logits,past_key_values. For reference, the inputs it received are input_ids,attention_mask."

Which checkpoint call should I use?
